syft/pkg/cataloger/homebrew/package.go

```diff
@@ -0,0 +1,41 @@
+package homebrew
+
+import (
+	"github.com/anchore/packageurl-go"
+	"github.com/anchore/syft/syft/file"
+	"github.com/anchore/syft/syft/pkg"
+)
+
+func newHomebrewPackage(name, version, desc, homepage string, locations file.LocationSet) pkg.Package {
+	p := pkg.Package{
+		Name:      name,
+		Version:   version,
+		Type:      pkg.HomebrewPkg,
+		Locations: locations,
+		Language:  pkg.Ruby,
```

Comment:
I don't think Ruby is the right thing for this. The packages themselves are defined in ruby, but they don't really contain ruby. This is probably best to leave blank. Language could be used in Grype for example, and would indicate to look for Ruby-related GHSA matches, but these would not be specified there.